stackit/stackit_core.py

```diff
@@ -77,7 +77,17 @@ def searchTerm(term, tags):
             if(count % NUM_RESULTS == 0):
                 focusQuestion(questionLogs, count)
         j+=1
-
+    if len(questions) == 0:
+        response = raw_input("No questions found matching your input. Try broader search terms. \nPress s to search again, b to enter browser or x to exit: ")
+        if response == "x":
+            sys.exit()
+        elif response == "b":
+            webbrowser.open("https://stackoverflow.com")
+        elif response == "s":
+            searchTerms = raw_input("Enter search terms: ")
+            os.system('stackit %s' % searchTerms)
```

Comment:
I think the os.system call is a bit messy, and we could do this more cleanly.

@vickiniu could we do a recursive call to the `searchTerm` method here? Skips all the parsing stuff, and keeps it all in one process. Line 88 would become `searchTerm(searchTerms, ""), unless we integrate tagging functionality.